pc/storage/kdtree/kdtree_test.go

```diff
@@ -122,601 +122,608 @@ func kdtreeDeepExpectEqual(t *testing.T, a, b *KDTree) bool {
 }
 
 func TestKDtree(t *testing.T) {
-	it := createTestPointCloud(t)
-	kdt := New(it)
+	for _, minDist := range []float32{0, 0.001} {
```

Comment:
[optional] Some of the tests do not seem to be affected by `minDistSq` (e.q. tree construction, range, delete). Maybe they could be defined outside of the for loop.